### PR TITLE
[@types/vimeo__player] Update typings to v2.10.0

### PR DESCRIPTION
**Pull request by kou029w**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint @vimeo/player`.
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
  - documentation: https://github.com/vimeo/player.js/blob/v2.10.0/README.md
  - pull request: https://github.com/vimeo/player.js/pull/455
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/vimeo__player/index.d.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
-// Type definitions for @vimeo/player 2.9.1
+// Type definitions for @vimeo/player 2.10.0
```
Comment (peterblazejewicz) [inline types/vimeo__player/index.d.ts:1]:
```suggestion
// Type definitions for @vimeo/player 2.10
```
https://github.com/definitelyTyped/DefinitelyTyped/#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@kou029w thx!

**Review by peterblazejewicz — APPROVED** [review]

**Review by terrymun — CHANGES_REQUESTED** [review]
The file `vimeo__player-tests.ts` is not updated with the new `getMuted` and `setMuted` methods that the definitions file now support. Please look into including them as well.

**Comment by peterblazejewicz** [comment]
@terrymun the tests ships with usage that seems tb taken stright from the docs, what else should be there?

**Comment by terrymun** [comment]
@peterblazejewicz My bad: for some reason when I looked at the diff, I only saw a single file being changed: changes are, I was looking at a specific commit :)

**Review by terrymun — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]

**Pull request merged**